src/posts/articles/impact-of-tax-changes-2025-2026.md

```diff
@@ -0,0 +1,241 @@
+Four tax reforms will take effect in April 2025. Using the PolicyEngine UK tax-benefit microsimulation model, we analysed these reforms, which will increase taxes by an average of £1,112 per household.
+
+This analysis, requested and [published](https://insider.iea.org.uk/p/new-analysis-the-cost-of-tax-hikes) by the Institute for Economic Affairs, featured in the Daily Mail's [front-page article](https://www.dailymail.co.uk/news/article-14548907/Labours-tax-raids-cost-families-week.html) on the 28th March 2025, in two [Express](https://www.express.co.uk/news/politics/2034186/rachel-reeves-tax-increase-cost) [articles](https://www.express.co.uk/finance/personalfinance/2034211/rachel-reevesl-tax-hikes-cost-british-families), and two [other](https://www.dorsetecho.co.uk/news/25047728.tax-hikes-will-cost-british-families-1-000-per-year-average/) [outlets](https://londonlovesbusiness.com/labours-tax-hikes-will-cost-british-families-over-1000-per-year-on-average/).
+
+## Policy changes
+
+From a combination of temporary reforms expiring and new tax reforms taking effect
```

Comment:
Can you also please specify when these reforms were announced in more detail, like in the third bullet? (I'd genuinely like to know without looking it up)
```suggestion
From a combination of temporary reforms expiring and new tax reforms taking effect:
```